plutus-tx/test/Plugin/Spec.hs

```diff
@@ -407,6 +525,9 @@ integer = plc @"integer" (1::Integer)
 free :: CompiledCode Bool
 free = plc @"free" (True && False)
 
+negativeInt :: CompiledCode Int
+negativeInt = plc @"negativeInt" (-1 :: Int)
```

Comment:
@michaelpj, note that we can't compile negative ints.

Reply:
We probably should be able to.